spec.html

```diff
@@ -489,15 +489,20 @@ <h1>Iterator.prototype.take ( _limit_ )</h1>
           1. Let _integerLimit_ be ! ToIntegerOrInfinity(_numLimit_).
           1. If _integerLimit_ &lt; 0, throw a *RangeError* exception.
           1. Let _closure_ be a new Abstract Closure with no parameters that captures _iterated_ and _integerLimit_ and performs the following steps when called:
+            1. If _integerLimit_ is 0, then
```

Comment:
For consistency, `bounded = iter.take(0);` should close `iter` _immediately`, not waiting until the `bounded.next()` call. As currently written it does wait for that call. This would need to be outside of `_closure_` to have the immediately-closing behavior.